.github/workflows/reusable-backend-integration.yml

```
@@ -0,0 +1,88 @@
+  workflow_call:
+    inputs:
+      python-version:
+        required: true
+        type: string
+      image-tag:
+        required: true
+        type: string
+    outputs:
+      success:
+        description: "Whether the tests passed"
+        value: ${{ jobs.postman-tests.result }}
```

actionlint (1.7.11)

[error] property "result" is not defined in object type {outputs: {}}

(expression)